instagram/src/App.js

```diff
@@ -0,0 +1,138 @@
+import React, { Component } from "react";
+import moment from "moment";
+import dummyData from "./dummy-data";
+import SearchBar from "./components/SearchBar/SearchBar";
+import PostContainer from "./components/PostContainer/PostContainer";
+import "./css/main.css";
+
+class App extends Component {
+  constructor() {
+    super();
+    this.state = {
+      data: [],
+      newCommentText: "",
```

Comment:
be careful about lifting this out of the individual commentSections to APP -- with only one instance of this data to work off of, each comment section will display the typed text simultaneously

Reply:
ooh - didn't notice that - I'll try to fix it today.